cookbook/bundles/best_practices.rst

```diff
@@ -330,17 +311,23 @@ Retrieve the configuration parameters in your code from the container::
 
     $container->getParameter('acme_hello.email.from');
 
-Even if this mechanism is simple enough, you are highly encouraged to use the
-semantic configuration described in the cookbook.
+Even if this mechanism is simple enough, you should consider using the more
+advanced :doc:`semantic bundle configuration </cookbook/bundles/configuration>`.
 
-.. note::
+Services
+--------
+
+If the bundle defines services, they must be prefixed with the bundle alias.
+For an AcmeBlogBundle for instance, all services must be prefixed with
+``acme_blog_``.
 
-    If you are defining services, they should also be prefixed with the bundle
-    alias.
+In addition, services not meant to be used in the application, should be defined
+as private.
```

Comment:
we could add a `.. seealso::` directive to the article about loading services using the DI extension